src/components/Header/MobileNav/index.module.scss

```diff
@@ -81,7 +81,7 @@
   all: unset;
   width: 100%;
   height: 100%;
-  padding-top: var(--header-height);
+  padding-top: calc(var(--header-height) * 1.5);
```

Comment:
There's a `var(--page-padding-top)` that is probably better choice here but no need to change this now.